scripts/earthengine/process_events.py

```diff
@@ -1034,29 +1136,239 @@ def read_active_events(self, filename: str):
         logging.info(
             f'Loaded {len(active_events)} events from file: {filename}')
 
+    def write_events_place_svobs(
```

Comment:
This is an incredibly long function :). I would recommend adding more comments on the high level flow

Reply:
Added a comment and broke it down into smaller functions.